app/screens/home/channel_list/servers/servers_list/server_item/server_item.tsx

```diff
@@ -59,7 +59,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
     badge: {
         left: 18,
         top: -5,
-        borderColor: theme.centerChannelBg,
+        borderWidth: 1,
```

Comment:
@asaadmahmood your request to make this a 1px, but the figma you sent is 2px.

@larkox I believe we should set up the proper protocol if we want to move from 2px to 1px.